pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -555,6 +561,7 @@ def create_server_instance(connection_name):
 
     pywbem_server = None
     resolved_default_namespace = default_namespace or DEFAULT_NAMESPACE
+    # TODO: Do i need this
```

Comment:
This is supposed to apply the default when the option is unspecified. I think it was incorrect even for the old boolean timestats (because it did not look for None as a condition to use the default value), and is probably still incorrect now.

Reply:
Since the default when the operation is not defined is None and this states either timestats if not None or None.  So actually corect bur redundent I believe.  Therefore removed DEFAULT_TIMESTATS and resolved_timestats completely.

Reply:
DONE